src/style.css

```diff
@@ -1 +1,57 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+  object-fit: none;
```

Comment:
Do you need these properties `object-fit: none` and `object-position: top`? They do not seem to be necessary here.